testing/data-migration/migrateData.sh

```diff
@@ -11,25 +11,27 @@ set -euo pipefail
 function upload_data_set () {
   data_set_file="$framework_output/$data_id.json"
   response_file="$framework_output/$data_id.response"
-  curl -X POST "https://$target/api/data/$framework?bypassQa=true" -H 'accept: application/json' -H "Authorization: Bearer $target_token" \
```

Comment:
Just to see if i have understood the changes correctly.
There is a config file now which defines the expected inputs; some more error logging;  possibility to select only datasets that are active or all datasets. Is that summary correct?

Reply:
to my understading - yes. Siegfried has the full insight here

Reply:
Siegfried confirmed that these are the changes.

Reply:
Ok, cool. Closing this here